### ISSUE TITLE
Some widgets do not scale on changing layout width and height

### ISSUE THREAD

**Issue by ITCCodeDev** [dji-sdk/Mobile-UILibrary-Android#11]
The `dji.ui.widget.BatteryWidget` and the `dji.ui.widget.ConnectionWidget` do not scale up on increasing the layout width and height (even while maintaining the expected aspect ratio). 
The connection widget icon stays the same size and just repositions to the top of the new dimensions, while the battery widget icon scales up as expected but the battery widget text does not scale and just repositions to the top of the widget. Here are the images for the same - 
Correctly displaying widgets when layout_height is set to 25dp -
![correct_25dp](https://user-images.githubusercontent.com/24898794/27154540-6341e14e-5123-11e7-801b-75dc4269bcd3.jpg)

Incorrectly displaying widgets when layout_height is set to 50dp - 
![incorrect_50dp](https://user-images.githubusercontent.com/24898794/27154541-6344c45e-5123-11e7-9f32-46b17c752b9b.jpg)

I had the parameters for both of these set as follows (following the recommended aspect ratios) - 
```
          <dji.ui.widget.BatteryWidget
            android:layout_width="128dp"
            android:layout_height="50dp"/>
        <dji.ui.widget.ConnectionWidget
            android:layout_width="50dp"
            android:layout_height="50dp"/>
```
I am using the Android UI Library 4.1.1 with the DJI Inspire 1 on a Nexus 7 tablet running Android 6.0.1.
The Android Studio version is 2.3.3.

**Comment by ITCCodeDev** [comment]
Is there any update on this? It is still an issue in the 4.2 version of the UI Library. It is very difficult to position the battery widget appropriately now.

**Comment by oliverou** [comment]
Hi @ITCCodeDev, thanks for the feedbacks. We have addressed the issue and will fix it in the upcoming UILibrary release.

**Comment by talobin** [comment]
This problem was fixed in version 4.3.2.
Will go ahead and close this issue.
@ITCCodeDev please feel free to reopen if the bug still exists.
